sdk/keyvault/azure-security-keyvault/azure/security/keyvault/aio/keys/samples/hello_world_async.py

```diff
@@ -0,0 +1,95 @@
+import datetime
+import asyncio
+import os
+import pytz
+from azure.security.keyvault.aio import KeyClient
+from azure.identity import AsyncDefaultAzureCredential
+from azure.core.exceptions import HttpResponseError
+
+# ----------------------------------------------------------------------------------------------------------
+# Prerequistes -
+#
+# 1. An Azure Key Vault-
+#    https://docs.microsoft.com/en-us/azure/key-vault/quick-create-cli
+#
+#  2. Microsoft Azure Key Vault PyPI package -
+#    https://pypi.python.org/pypi/azure-security-keyvault/
+#
+# 3. Microsoft Azure Identity package -
+#    https://pypi.python.org/pypi/azure-identity/
+#
+# 4. Set Environment variables AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET, VAULT_URL. [How to do this](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault/azure/security/keyvault/keys#createget-credentials)
+#
+# ----------------------------------------------------------------------------------------------------------
+# Sample - demonstrates the basic CRUD operations on a vault(key) resource for Azure Key Vault
+#
+# 1. Create a new RSA Key (create_rsa_key)
+#
+# 2. Create a new EC Key (create_rsa_key)
+#
+# 3. Get an existing key (get_key)
+#
+# 4. Update an existing key (set_key)
+#
+# 5. Delete a key (delete_key)
+# ----------------------------------------------------------------------------------------------------------
+async def run_sample():
+    # Instantiate a key client that will be used to call the service.
+    # Notice that the client is using default Azure credentials.
+    # To make default credentials work, ensure that environment variables 'AZURE_CLIENT_ID',
+    # 'AZURE_CLIENT_SECRET' and 'AZURE_TENANT_ID' are set with the service principal credentials.
+    VAULT_URL = os.environ["VAULT_URL"]
+    credential = AsyncDefaultAzureCredential()
+    client = KeyClient(vault_url=VAULT_URL, credential=credential)
+    try:
+        # Let's create an RSA key with size 2048, hsm disabled and optional key_operations of encrypt, decrypt.
+        # if the key already exists in the Key Vault, then a new version of the key is created.
+        print("\n1. Create an RSA Key")
+        key_size = 2048
+        key_ops = ["encrypt", "decrypt", "sign", "verify", "wrapKey", "unwrapKey"]
+        key_name = "rsaKeyName"
+        rsa_key = await client.create_rsa_key(key_name, size=key_size, hsm=False, key_operations=key_ops)
+        print("RSA Key with name '{0}' created of type '{1}'.".format(rsa_key.name, rsa_key.key_material.kty))
+
+        # Let's create an Elliptic Curve key with algorithm curve type P-256 and allow it to be processed in an HSM
+        # if the key already exists in the Key Vault, then a new version of the key is created.
+        print("\n1. Create an EC Key")
+        key_curve = "P-256"
+        key_name = "ECKeyName"
+        ec_key = await client.create_ec_key(key_name, curve=key_curve, hsm=True)
```

Comment:
I think we should show only `hsm=False` in these samples because HSMs require a premium vault and support scenarios more advanced than Hello World.